src/pipeline_configuration_spec.py

```diff
@@ -38,9 +39,13 @@ class RapidProTarget:
 @dataclass
 class PipelineConfiguration:
     pipeline_name: str
+    project_start_date: str
```

Comment:
We talked offline about the possibility of tidying this up into e.g. filters = [TimeFilter(...), UuidFilter(...)] etc. and structuring configuration so that we don't need to pass the entire configuration into each individual step. But I think these are not priorities so fine to add a todo for those for now

Reply:
Added a todo.